### PR TITLE
Fixes on stats

### LINKED ISSUE #34: Descriptors Loader shows >100% errors
image (1)

### PR DESCRIPTION
**Pull request by luisremis**
Fix #34

### CODE REVIEW

**Comment by gsaluja9** [comment]
> Fix #34

Since it takes some time to identify the scenario in which these corner cases lead to problems, Can some tests be added to simulate them?

**Comment by luisremis** [comment]
I don't have examples to reproduce this. I just reviewed the implementation and made some changes where I found potential issues. All I have is the customers reports and some situations where I encountered this in #34

**Comment by gsaluja9** [comment]
> I don't have examples to reproduce this. I just reviewed the implementation and made some changes where I found potential issues. All I have is the customers reports and some situations where I encountered this in #34

I was able to repro the negative values scenario. Please take a look and merge if changes make sense. We can discuss alternatives.

**Review by gsaluja9 — DISMISSED** [review]

File: aperturedb/ParallelQuery.py [hunk 1]
```diff
@@ -195,25 +205,29 @@ def print_stats(self):
...
+
+            print(f"Avg Query time (s): {mean}")
+            print(f"Query time std: {std}")
+            print(f"Avg Query Throughput (q/s): {tp}")
 
-            msg = "(" + self.type + "/s):"
-            print("Overall throughput", msg,
-                  self.total_actions / self.total_actions_time)
+            i_tp = self.total_actions / self.total_actions_time
+            # print(f"Overall insertion throughput ({self.type}/s): {i_tp}")
```
Comment (luisremis) [inline aperturedb/ParallelQuery.py:224]:
```suggestion
```

File: aperturedb/ParallelLoader.py [hunk 2]
```diff
@@ -27,32 +23,41 @@ def ingest(self, generator, batchsize=1, numthreads=4, stats=False):
...
-                      self.error_counter / total_queries_exec)
-
-            print("Total inserted elements:", inserted_elements)
+                err_perc = 100 * self.error_counter / total_queries_exec
+                print(f"Total errors encountered: {self.error_counter}")
+                print(f"Errors (%): {err_perc}")
+
+            # TODO this does not take into account that the last
+            # batch may be smaller than batchsize
+            # inserted_elements = self.total_actions - self.error_counter * self.batchsize
```
Comment (luisremis) [inline aperturedb/ParallelLoader.py:60]:
```suggestion
```
